packages/common/src/hooks/useQueryParams/usePagination.stories.tsx

```diff
@@ -1,43 +0,0 @@
-// import React from 'react';
```

Comment:
I was already wondering the last PR what are these stories?

Reply:
the stories are a way to showcase / demonstrate components using storybook. To run locally, use `yarn storybook` or for the slightly easier version.. it is hosted on demo: https://demo-open.msupply.org:6006/

The idea is to show the scope of available components, which helps prevent developers creating duplicates, and to show the various options available within components.

Reply:
thanks!